packages/desktop/components/menus/AccountActionsMenu.svelte

```diff
@@ -52,17 +51,22 @@
             id: PopupId.Confirmation,
             props: {
                 variant: 'danger',
-                title: localize('popups.deleteAccount.title', { values: { name: $selectedAccount?.name } }),
+                title: localize('popups.deleteAccount.title', { values: { name: account.name } }),
                 alert: { variant: 'warning', text: localize('popups.deleteAccount.hint') },
                 confirmText: localize('actions.delete'),
                 onConfirm: async () => {
-                    await checkActiveProfileAuth(
-                        async () => {
-                            await deleteAccount($selectedAccount?.index)
-                            closePopup()
-                        },
-                        { stronghold: true }
-                    )
+                    try {
+                        await checkActiveProfileAuthAsync()
+                    } catch {
+                        return
```

Comment:
This doesn't work automatically  like this. I'm stuck on the following screen for ledger profiles due to the return condition: Once the user authenticates, it has to automatically delete the account.
![image](https://github.com/bloomwalletio/bloom/assets/45079109/2e649e30-8466-41dd-8f4f-1f74251feb98)